### PR TITLE
TimePicker (mobile): Support the 24h format type

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Issue] https://github.com/Samsung/TAU/issues/897
[Problem] Need to implement the TimePicker(24h)
[Solution] Add an example and styles for the TimePicker (24h)

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Review by singa2000 — APPROVED** [review]
It's good

File: src/css/profile/mobile/common/timepicker.less [hunk 1]
```diff
@@ -22,6 +22,28 @@
...
 	}
 
+	&[data-format="24"] {
+		.ui-time-picker-container {
+			width: 30%;
+
+			&-hour::after {
+				content: ":";
+				height: 164 * @px_base;
+				width: 40 / 3%;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/timepicker.less]: It looks incorrect... is this works?

Comment (HunseopJeong) [reply src/css/profile/mobile/common/timepicker.less]: Yes,, it works,, Wouldn't it be strange to use 'calc'?
According to guide(4-15), Time Spin has 30% for the width. So, the remaining area(40%) is divided into three.

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/timepicker.less]: I don't see the `calc` and I don't understand this calculation.

Comment (HunseopJeong) [reply src/css/profile/mobile/common/timepicker.less]: Yes, it is just the 40 over 3. 40/3 = 13.333333 %.

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/timepicker.less]: maybe you wanted to write:
`calc(40% / 3)`

Comment (HunseopJeong) [reply src/css/profile/mobile/common/timepicker.less]: Right.
Done.

File: src/css/profile/mobile/common/timepicker.less [hunk 2]
```diff
@@ -22,6 +22,28 @@
...
 	}
 
+	&[data-format="24"] {
+		.ui-time-picker-container {
+			width: 30%;
+
+			&-hour::after {
+				content: ":";
+				height: 164 * @px_base;
+				width: calc(40% / 3);
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/timepicker.less]: I'm not sure your intention. You wanted calc to be done in `less` or `css`?
For `less` will be enough `40% / 3` but in css `~"calc(40% / 3)"`

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/timepicker.less]: I think it was typo

Comment (HunseopJeong) [reply src/css/profile/mobile/common/timepicker.less]: Okay. I will recover it to done it in `less`.

File: src/css/profile/mobile/common/timepicker.less [hunk 3]
```diff
@@ -22,6 +22,28 @@
...
 	}
 
+	&[data-format="24"] {
+		.ui-time-picker-container {
+			width: 30%;
+
+			&-hour::after {
+				content: ":";
+				height: 164 * @px_base;
+				width: 40% / 3;
```
Comment (DohyungLim) [inline src/css/profile/mobile/common/timepicker.less:32]: Is the width of the specification document divided by 3?

I'm worried if the value after the decimal point is fine even if the width is not correct.

**Review by DohyungLim — COMMENTED** [review]
1 comment

File: src/css/profile/mobile/common/timepicker.less (hunk 3 above)
Comment (HunseopJeong) [reply src/css/profile/mobile/common/timepicker.less:32]: I think there is no way to calculate the correct size for the width.

**Pull request merged**